designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/BaseEndpointDefinitions.scala

```diff
@@ -86,9 +87,24 @@ object BaseEndpointDefinitions {
 
 }
 
-sealed trait SecurityError
+sealed trait NuError
+case class NuException(nuError: NuError) extends Exception
```

Comment:
I don't think it's a good idea. IMO we should get rid of the concept of throwing exceptions in case of non-happy path of Nu business logic.

So, e.g. method like this:
```
def check(scenarioName: ProcessName, permission: Permission)(implicit user: LoggedUser): Future[ProcessId] 
```

should explicitly say what the potential bad result types are. E.g. using `Either`

Reply:
Done